src/history/hs_rec.c

```diff
@@ -474,6 +477,9 @@ __wt_hs_insert_updates(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_MULTI *mult
                 break;
         }
 
+        if (newest_hs == NULL)
```

Comment:
Can we add a comment on why we are skipping the rest of the loop here? Is it because there is no full update to be written since `newest_hs` has not been found?